core/domain/role_services_test.py

```diff
@@ -48,6 +49,15 @@ def test_get_all_actions(self) -> None:
 
     def test_action_allocated_to_all_allowed_roles(self) -> None:
         role_actions = role_services.get_role_actions()
-
+        
         self.assertItemsEqual(  # type: ignore[no-untyped-call]
             list(role_actions), feconf.ALLOWED_USER_ROLES)
+
+    def test_log_role_query(self):
+        role_services.log_role_query('TEST_USER',
+            feconf.ROLE_ACTION_ADD, role='GUEST')
```

Comment:
When breaking a line inside of parentheses, please always break right after the opening parenthesis.

Reply:
Done